packaging/ubuntu-16.04/rules

```diff
@@ -209,7 +207,7 @@ endif
 	$(MAKE) -C data all
 
 	# build squashfuse and rename to snapfuse
-	(cd vendor/github.com/snapcore/squashfuse/src && mkdir -p autom4te.cache && ./autogen.sh --disable-demo && ./configure --disable-demo && make && mv squashfuse_ll snapfuse)
+	(cd _build/src/$(DH_GOPKG)/vendor/github.com/snapcore/squashfuse/src && mkdir -p autom4te.cache && ./autogen.sh --disable-demo && ./configure --disable-demo && make && mv squashfuse_ll ../../../../vendor/snapfuse)
```

Comment:
sadly this does not seem to work, either with this change or with what's in master, would appreciate some help on what is going on with how and why we are building squashfuse/snapfuse like this

Reply:
Do you have logs for the build failure at hand?